client/components/theme-collection/index.tsx

```diff
@@ -65,19 +65,19 @@ export default function ThemeCollection( {
 							prevEl: '.theme-collection__carousel-nav-button--previous',
 						},
 						threshold: 5,
-						slideToClickedSlide: true,
+						slideToClickedSlide: false,
+						rewind: true,
```

Comment:
The rewind property seems to be causing the arrows of the carousel to always be enabled since we can always navigate forward and backward now.

So now the carousel can never be in this state:
![Captura de pantalla 2023-10-18 a las 8 52 19](https://github.com/Automattic/wp-calypso/assets/1989914/9bbc594d-ec4f-4688-92e4-81a16583c979)

Was `rewid: true` needed for something regarding the movement of the items in the carousel?

Reply:
> Was `rewid: true` needed for something regarding the movement of the items in the carousel?

Nay, I just thought it was a better UX, but forgot to mention it in the PR description. (Adding it now)

Reply:
There's nothing to do, really. 🙂 

The Swiper library controls the disabled logic by applying a `disabled` attribute.
The opacity is controlled by the Core `Button` component, based on the `disabled` attribute.

Reply:
There's some CSS:
```
&.swiper-button-disabled {
			pointer-events: none;
		}
```

Reply:
We could cleanup `transition: opacity 0.3s linear;` and `&.swiper-button-disabled {
			pointer-events: none;
		}` from the `theme-collection__carousel-nav-button` class in `theme-collection-style.scss`

Reply:
All right, all right. 😄

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/83113/commits/b01676b09fea2706b543462f0a045da1ecbe48fc